### ISSUE TITLE
Request: Is it posible to add a comma between prompts.

### ISSUE THREAD

**Issue by sbuldan** [fa0311/danbooru-to-prompt#5]
It is realy hard to read the prompts when you paste it to generator. It would be awesome If we could have an option to add comma between prompts.

**Comment by fa0311** [comment]
@sbuldan 
If I remember correctly, this feature was removed in the past

because commas affect Tokens.
I haven't touched Stabel Diffusion for a while, so I don't know if it still affects me.

Is there any benefit in terms other than readability?
Also, overall, do commas have a positive impact?

**Comment by sbuldan** [comment]
Yes other than readability, it can prevent misunderstanding of a prompt. For example if you give out "blue_eyes closed_window" as a prompt, it gives results with closed eyes, since the comma is tokenized it prevents that situation. Grid 40 is with a comma and grid 41 is without.

![grid-0040](https://github.com/fa0311/danbooru-to-prompt/assets/88350108/b0531ed6-bc7b-4867-8592-6cba1d6f6739)
![grid-0041](https://github.com/fa0311/danbooru-to-prompt/assets/88350108/69168637-d65c-4928-8112-efaefa0b7710)
